connectivity/connectivity-demos-test/src_test/com/axonivy/connectivity/rest/TestRestClient.java

```diff
@@ -0,0 +1,26 @@
+package com.axonivy.connectivity.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.environment.IvyTest;
+
+@IvyTest
+public class TestRestClient
```

Comment:
Instead of using the Ivy direct in the test write a test for a productive class that uses Ivy. For example you can write a test for com.axonivy.connectivity.rest.client.PersonClient. This class uses Ivy.rest in its methods.

A user will also write IvyTests to test its productive classes that uses Ivy. He will not write a test and uses Ivy itself except to do some fixtures